dz2.12/src/test/java/ru/skypro/dz2/_2/ApplicationTests.java

```diff
@@ -0,0 +1,90 @@
+package ru.skypro.dz2._2;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import ru.skypro.dz2._2.exceptions.EmployeeAlreadyAddedException;
+import ru.skypro.dz2._2.exceptions.EmployeeNotFoundException;
+import ru.skypro.dz2._2.exceptions.EmployeeStorageIsFullException;
+import ru.skypro.dz2._2.model.Employee;
+import ru.skypro.dz2._2.service.EmployeeService;
+import ru.skypro.dz2._2.service.EmployeeServiceImpl;
+import ru.skypro.dz2._2.service.SalaryEmployeeService;
+import ru.skypro.dz2._2.service.SalaryEmployeeServiceImpl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class ApplicationTests {
+
+    @Autowired
+    EmployeeServiceImpl serviceImpl;
```

Comment:
```suggestion
    EmployeeService serviceImpl;
```
инъекцию надо делать по интерфейсу